.gitignore

```diff
@@ -0,0 +1,35 @@
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
```

Comment:
May also consider whether some of the other hidden files need to be committed or not like .gitignore itself? Just for comparison here are some of the . files I included in the project   if that helps.  
 
<img width="1001" alt="Screenshot 2022-04-25 at 08 01 57" src="https://user-images.githubusercontent.com/68547574/165038467-aaae4c6c-8b41-496b-80ea-03c422823f02.png">

Reply:
Sorry I meant that some of the files that are in the PR could be included in .gitignore so we are limiting what files go up to github Eg .gitignore itself and .gitattributes. Not important at this stage but I found it helpful to start thinking about what things to make public early so it was easier to keep the more important env files etc hidden down the line. Does that make sense?

Reply:
Oh sorry I misunderstood! Yes it makes sense 🙂. I wonder if to add the `.gitignore `itself is good practice. As per https://stackoverflow.com/questions/767147/how-can-i-stop-gitignore-from-appearing-in-the-list-of-untracked-files

> The `.gitignore` file should be in your repository, so it should indeed be added and committed in, as git status suggests. It has to be a part of the repository tree, so that changes to it can be merged and so on.
> 
> So, add it to your repository, it should not be gitignored.

Reply:
You're right I just did the google check and `.gitattributes` and `.gitignore` should be included - ignore me :)

Reply:
I've added some files following https://github.com/github/gitignore/blob/main/Ruby.gitignore 👍 thank you Kate!